### ISSUE TITLE
Zend Framework update

### ISSUE THREAD

**Issue by userdevel** [Xtraball/Siberian#670]
Last version here:
- https://github.com/zendframework/zendframework
- very old 1.12.x, last 1.12.20: https://github.com/zendframework/zf1/releases

**Comment by Slicit** [comment]
ZF1 to ZF2+ involves a lot of changes it's not possible to upgrade without recoding everything.

I'm not saying we won't at some point migrate, but we won't upgrade, we have to keep version safe.

Moreover, as said many times, most of our base tools, are all forked and custom, EVEN ZF1 is highly customised to our needs.

**Comment by userdevel** [comment]
For ZF1, it is easy, there is a ticket with all changes done (some are taken from next ZF1 versions):
- https://github.com/Xtraball/Siberian416/issues/15 job done by @Neustradamus

**Comment by Slicit** [comment]
The mentionned update will break Siberian, and will require lot of "side" fixes, we already tried to update it to the latest 1.x version in the past, and we can't.

There is two reasons:
1. Our ZF1 is custom, so not 100% the original version it's based on
2. Latest updates after the one we had, have breaking changes in sessions & some other things.

--
To finish, our ZF1 works as expected with our Siberian, and is updated to work with PHP 7.3, we do not need to upgrade it, there is no reason!

And if we have more needs, the way to go is to create a new "Siberian" class/library, we don't want to depend on ZF updates and changes.

We take what we need, and we work with it, we made it really stable, that's how you want to work when dealing with an unknown number of production installations, you choose one good version and only apply security patches.

When you run a Wordpress in production you never update as it goes, as long as there is no security risk or something not working at ALL.
